### ISSUE TITLE
Channel order sorting

### ISSUE THREAD

**Issue by francisbegbie** [yuliskov/SmartTube#3437]
Currently the channels can be sorted in this order -> New items (highlighted) and rest in alphabetical order.

Please allow user to change to ->  New items (highlighted) and rest in newest order.

**Comment by DragonWolf5589** [comment]
Yes please! Mine used to do this but seems to be mixed and all over place past 6 months.
